src/dispatch/incident/models.py

```diff
@@ -104,7 +104,8 @@ def last_status_report(self):
     @staticmethod
     def _status_time(mapper, connection, target):
         if target.status == IncidentStatus.stable:
-            target.stable_at = datetime.utcnow()
+            if not target.stable_at:
```

Comment:
@mvilanova this fixes the immediate issue, but I'm inclined to move this logic to the flows itself (this was a tricky one to track down). Thoughts?

Reply:
Shouldn't `update` [here](https://github.com/Netflix/dispatch/blob/develop/src/dispatch/incident/views.py#L94) take care of updating the time?